gobblin-runtime/src/main/java/org/apache/gobblin/runtime/api/MysqlMultiActiveLeaseArbiter.java

```diff
@@ -221,6 +237,31 @@ private void initializeConstantsTable() throws IOException {
     }, true);
   }
 
+  /**
+   * Periodically deletes all rows in the table with event_timestamp older than the retention period defined by config.
+   */
+  private void runRetentionOnArbitrationTable() {
+    while (true) {
+      try {
+        Thread.sleep(10000);
```

Comment:
tip: (same as if you use scheduled TP executor) - set sleep in time-esque values (such as 60).  also, 10s looks WAY too frequent, given a lease may itself last for minutes!  maybe we try this every six hours (4/daily)?